python/jupyter_suggestions_core/schema/commands.json

```diff
@@ -0,0 +1,14 @@
+{
+  "title": "@jupyter-suggestions/jupyter-suggestions-core",
```

Comment:
Is the `@jupyter-suggestions` scope a placeholder, or should we go with something like this for the final package name?

If so I guess it would be fine to already create the org on `npm`. Otherwise we could also use the `@jupyter` or `@jupyterlab` scopes later.

Reply:
you are right, I'm going to switch to `@jupyter`

Reply:
Do we need to ask for permission to publish packages under this org?